backend/curriculum_tracking/serializers.py

```diff
@@ -177,8 +177,14 @@ class Meta:
             "can_force_start",
             "open_pr_count",
             "oldest_open_pr_updated_time",
+            "users_that_reviewed_since_last_review_request",
         ]
 
+    users_that_reviewed_since_last_review_request = serializers.SerializerMethodField("get_users_that_reviewed_since_last_review_request")
+
+    def get_users_that_reviewed_since_last_review_request(self, instance):
+        return models.AgileCard.get_users_that_reviewed_since_last_review_request(instance)
```

Comment:
why not:

`return instance.get_users_that_reviewed_since_last_review_request()`

instance should be an AgileCard instance